src/trackCopiedText.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Tracking text copied by the user.
+ * 
+ * Adds event listener
+ */
+
+ export default (subscribe) => {
+    window.addEventListener("copy",function(e){
```

Comment:
use array function in this file

Reply:
when you use addEventListener, callback function should be named function like:

```
window.addEventListener('copy', processCopyEvent)
```

Reply:
Done